Rakefile

```diff
@@ -0,0 +1,12 @@
+require 'rototiller'
+require "bundler/gem_tasks"
```

Comment:
@er0ck The 'bundler/gem_tasks' require adds tasks for 'build', 'install', and 'release'.  I am not sure but I believe that the Gem pipelines in Jenkins may use it.